src/sql/workbench/contrib/query/browser/queryEditor.ts

```diff
@@ -254,25 +246,44 @@ export class QueryEditor extends BaseEditor {
 		return this._listDatabasesActionItem;
 	}
 
-	private setTaskbarContent(): void {
-		// Create HTML Elements for the taskbar
+	private setTaskbarContent(input: QueryEditorInput): void {
+		// Remove current actions from the taskbar
+		while (this.taskbar.length() > 0) {
+			this.taskbar.pull(0);
+		}
+
 		const separator = Taskbar.createTaskbarSeparator();
 		let content: ITaskbarContent[];
 		const previewFeaturesEnabled = this.configurationService.getValue('workbench')['enablePreviewFeatures'];
-		if (previewFeaturesEnabled) {
-			content = [
-				{ action: this._runQueryAction },
-				{ action: this._cancelQueryAction },
-				{ element: separator },
-				{ action: this._toggleConnectDatabaseAction },
-				{ action: this._changeConnectionAction },
-				{ action: this._listDatabasesAction },
-				{ element: separator },
-				{ action: this._estimatedQueryPlanAction }, // Preview
-				{ action: this._toggleSqlcmdMode }, // Preview
-				{ action: this._exportAsNotebookAction } // Preview
-			];
-		} else {
+		// TODOKusto: needs to be changed appropriately
```

Comment:
What does this mean? Is there a task to follow up on this? Please make sure that comments like this provide context to the issue for anyone reading it - right now I have very little context on why this is necessary or what needs to be done to fix it.